.github/workflows/develop-hosting-deploy.yml

```diff
@@ -0,0 +1,25 @@
+name: Develop hosting deploy
```

Comment:
The main reason was because they both require build steps now, but the builds happen in multiple working directories (functions in `functions/` and the UI at the top-level), so I found it easier to just duplicate the files. It is getting a little cluttered with all these deploy workflows, so I'm willing to try to consolidate them if we feel it's worth it

Reply:
Cool, I'm fine keeping it separate for now.